storage/scratchpad/src/sparse_merkle/counters.rs

```diff
@@ -0,0 +1,26 @@
+use libra_metrics::{register_histogram, Histogram};
```

Comment:
looks like somehow this is being pulled into LSR, you can import this through libra_secure_push_metrics without impacting it